src/lib.rs

```diff
@@ -419,7 +405,7 @@ pub fn optimize_from_memory(data: &[u8], opts: &Options) -> PngResult<Vec<u8>> {
     let optimized_output = optimize_png(&mut png, data, opts, deadline)?;
 
     if is_fully_optimized(original_size, optimized_output.len(), opts) {
-        eprintln!("Image already optimized");
+        warn!("Image already optimized");
```

Comment:
```suggestion
        info!("Image already optimized");
```

This probably makes more sense at `info`

Reply:
Yeah, I was on the fence because I thought that someone might want it even if they set logging level to exclude `info` (to `warn`), but makes sense.